docs/content/tutorial/step_02.ngdoc

```diff
@@ -49,128 +50,150 @@ __`app/index.html`:__
 </html>
 ```
 
-We replaced the hard-coded phone list with the {@link ng.directive:ngRepeat ngRepeat directive}
-and two {@link guide/expression Angular expressions}:
+We replaced the hard-coded phone list with the {@link ng.directive:ngRepeat ngRepeat} directive and
+two {@link guide/expression Angular expressions}:
```

Comment:
At first, I styled them as code, but it turned out it was very hard to distinguish clickable code (links) from plain code, so I switched to rely on non-code link styling for clickable/linkable stuff. I will use the simpler form though (`{@link ngRepeat ngRepeat}`).